### ISSUE TITLE
[Accessibility] Command history is not readable in terminal input field via Up/DownArrow

### ISSUE THREAD

**Issue by vscodenpa** [vscodenpa/testissues#15738]
CC -meganrogge

Tested with NVDA and JAWS on Windows 11.

-- Reproducible Steps

1. Load screen reader.

1. Launch terminal.

1. Type `echo hello` <enter>.

1. Type `echo world` <enter>.

1. In terminal input field, press UpArrow and DownArrow to navigate command history and see if you can hear them.

* Notes: Nothing is heard. Screen reader keeps saying "blank."

-- Expected Behavior

Command history needs to be accessible and readable to screen reader.

-- System Info

Version: 1.75.0-insider (system setup)
Commit: 0a6c277e4faa44a6b6eb82ddca8bfeccf7fecef6
Date: 2023-01-24T05:22:55.489Z
Electron: 19.1.9
Chromium: 102.0.5005.194
Node.js: 16.14.2
V8: 10.2.154.23-electron.0
OS: Windows_NT x64 10.0.22621
Sandboxed: Yes
